BreadPlayer.Framework/Messenger/MessageTypes.cs

```diff
@@ -14,7 +14,9 @@ public enum MessageTypes
         MSG_ADD_ALBUMS,
         MSG_PLAYLIST_LOADED,
         MSG_UPDATE_SONG_COUNT,
-        MSG_SEARCH_STARTED
+        MSG_SEARCH_STARTED,
+        MSG_ADD_STOP_SONG
```

Comment:
Make the `MessageType` name a little less inconspicuous. Something like `MSG_STOP_AFTER_SONG` or some such.